src/commonMain/kotlin/fr/acinq/lightning/payment/IncomingPaymentHandler.kt

```diff
@@ -222,7 +223,7 @@ class IncomingPaymentHandler(val nodeParams: NodeParams, val walletParams: Walle
                             pending[paymentPart.paymentHash] = payment
                             return ProcessAddResult.Pending(incomingPayment)
                         }
-                        payToOpenMinAmount is MilliSatoshi && payment.amountReceived < payToOpenMinAmount -> {
+                        payToOpenMinAmount != null && (payToOpenAmount < payToOpenMinAmount || payToOpenAmount < nodeParams.minFundingSatoshis) -> {
```

Comment:
nit: the warning that is logged below should also include `payToOpenAmount`

Reply:
Good catch, fixed in https://github.com/ACINQ/lightning-kmp/pull/283/commits/c2d8b93dfdb502629384fe3e1f0296e650671848